src/components/Pages/ActionsPage/StoryForm.js

```diff
@@ -272,15 +272,21 @@ class StoryForm extends React.Component {
       console.log("testimonial body", body)
       apiCall(Url, body).then((json) => {
         if (json && json.success) {
-          this.setState({
-            formNotification: {
-              icon: "fa fa-check",
-              type: "good",
-              text:
-                "Nicely done! Your story will be reviewed and published as soon as possible. Stay tuned!",
-            },
-          });
-          resetForm();
+			
+			if (this.props?.TriggerSuccessNotification) {
```

Comment:
@dim505 - still having issues with your tab settings?

Reply:
also note that for javascript, convention is to use camelCase variable names, except for class names.  So this should be triggerSuccessNotification.

Reply:
yeah sorry the tab spacing was set to 4